### ISSUE TITLE
Post detail page

### ISSUE THREAD

**Issue by Sooraj-beep** [UofA-Blueprint/AlzheimerCalgary#89]

**Comment by dilltran** [comment]
Is the post detail page the same as the activity card? Which is this referring to exactly?

**Comment by Sooraj-beep** [comment]
@dilltran
activity is a type of post but posts can be of type media or travel status. So I believe the post detail page is what shows up when you click on one of the posts.

@the-rooster @dimassudjito @loravocado Feel free to correct me if I am wrong

![Image](https://user-images.githubusercontent.com/55115712/202915095-80c84b57-b907-4580-affb-97e4b0355d94.png)

**Comment by dimassudjito** [comment]
Yes, I'm a bit unsure about how each post is connected to the post detail page, but there's this page which shows detail about the mood of client for an activity or any post type

![Image](https://user-images.githubusercontent.com/29143314/202922625-27ca1b11-b273-4796-a18b-0ccdbb838301.png)

**Comment by dilltran** [comment]
Gotcha. I was speaking to @jlsamson about the mood page which would be accessed through the "activity observation summary", and we think that it seems a little redundant to have a button redirect users to another page that has very limited information. I created another post detail page under my pages of the sketches that shows all of this stuff on the bottom of the initial description. It's still a little empty, so I could always cut the distance between the text if there isn't more stuff to include in the summary. Let me know what you guys think! 
![Post Detail Page](https://user-images.githubusercontent.com/115681016/202927942-07bcfe56-d4d3-4c22-a994-a6ff4562acd7.png)

**Comment by dilltran** [comment]
I took note of the comments and made appropriate adjustments. Let me know what you think!
